src/Symfony/Component/Uid/Command/GenerateUuidCommand.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Uid\Factory\UuidFactory;
+use Symfony\Component\Uid\Uuid;
+
+class GenerateUuidCommand extends Command
+{
+    protected static $defaultName = 'uuid:generate';
+    protected static $defaultDescription = 'Generates a UUID';
+
+    private $factory;
+
+    public function __construct(UuidFactory $factory = null)
+    {
+        $this->factory = $factory ?? new UuidFactory();
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputOption('time-based', null, InputOption::VALUE_REQUIRED, 'The timestamp, to generate a time based UUID: a parsable date/time string by the \DateTimeImmutable constructor. It must be greater than or equals to the UUID epoch (1582-10-15 00:00:00)'),
```

Comment:
`A parsable date/time string` would be enough for the description

Reply:
Should I remove the "it must be greater ..." part too?

Reply:
I kept the last part.